### DISCUSSION TITLE
[Proposal Review] API blueprint (Idea 4) and Schema Registry (Idea 7)

### DISCUSSION THREAD

**Discussion by pcgamer1** [postmanlabs/gsoc#35] [Ideas]
Hey! I am Sarthak Saxena, a 3rd year undergrad studying CSE at SRM Institute of Science and Technology, KTR, Chennai.

I am interested and looking forward to working on Idea 4 / 7.
I have drafted 2 proposals - 
[Proposal for Idea 4](https://docs.google.com/document/d/1OkeQ24vvxapVWhFUhW5s2P3f2juz7D6UDO8CyAj6Cgk/edit?usp=sharing)
[Proposal for Idea 7](https://docs.google.com/document/d/1ogbEC-oiiXdSRPezKtlzTw_nuOT8NwRLkORizmHiYZs/edit?usp=sharing)
Please have a look, Thanks.

**Comment by Sharath-Postman** [comment]
Thank you so much for writing in and it's my understanding that you have received reviews on both the proposals and in constant communication with the relevant mentor.

Cheers
